tools/fuzzer/main.c

```diff
@@ -76,7 +76,8 @@ static struct fuzz_platform *platform[] = {
 		&cht_platform,
 		&bsw_platform,
 		&hsw_platform,
-		&bdw_platform
+		&bdw_platform,
+		&imx8_platform
```

Comment:
Reminding you of the comment on Diana's original PR, this should end with a comma as well to prevent further two-line diffs.